Cargo.toml

```
@@ -20,7 +20,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "~0.20"
+tree-sitter = ">=0.22.0"
```

```suggestion
tree-sitter = ">=0.20.0"
```

since we're compatible with the older versions too